<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyMatchingAlignmentFormClass</class>
 <widget class="GUIFormBase" name="PropertyMatchingAlignmentFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertyMatchingAlignmentForm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>特徴面積比率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>特徴分散下限</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxCoverRate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxPickUpVar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCreateSpecified">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>選択アイテムの特徴領域を作成</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyMatchingAlignment.qrc">
     <normaloff>:/Qtres/Make.png</normaloff>:/Qtres/Make.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Layer</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ItemID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Area</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="toolButtonSpecified">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>特徴領域の表示</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyMatchingAlignment.qrc">
     <normaloff>:/Qtres/Area.png</normaloff>:/Qtres/Area.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>ブロックサイズ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBlockSize">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyMatchingAlignment.qrc"/>
 </resources>
 <connections/>
</ui>
